src/GraphQl/Resolver/Factory/CollectionResolverFactory.php

```diff
@@ -41,6 +41,10 @@ public function __construct(private readonly ReadStageInterface $readStage, priv
     public function __invoke(string $resourceClass = null, string $rootClass = null, Operation $operation = null): callable
     {
         return function (?array $source, array $args, $context, ResolveInfo $info) use ($resourceClass, $rootClass, $operation): ?array {
+            if (!isset($info->getFieldSelection()['paginationInfo']) && $operation->getPaginationClientPartial()) {
```

Comment:
I'm not sure if that's relevant to check `$operation->getPaginationClientPartial` but at least the behavior will follow the configuration

Reply:
It doesn't make sense to have `$operation->getPaginationClientPartial`.
If pagination client partial is enabled, the operation already has it.

Reply:
It does not make sense to check the `$operation->getPaginationClientPartial`.
GraphQL is always client-wise: if the user has not asked the full pagination info then we can safely enable the partial pagination, even if client partial is not enabled.
Also why checking `paginationInfo` and not the specific full pagination fields (`totalCount`, `lastPage`)?